userdocs/diagnostics/diagnostics.md

```diff
@@ -4,4 +4,84 @@
    @TechnologyRoot
 }
 
-Documentation on diagnostics emitted by the compiler.
\ No newline at end of file
+Documentation on diagnostics emitted by the compiler and settings that control them.
+
+
+## Diagnostics
+
+Swift diagnostics are classified into errors and warnings. Warnings can only be silenced within
+source in an intentional manner, eg. adding `_ =` for an unused function result.
+
+Some diagnostics have extra documentation, these include a `[#Name]` inline and reference to this
```

Comment:
```suggestion
Some diagnostics have extra documentation. These include a `[#Name]` inline and reference to this
```

Reply:
Thanks, that was indeed a bit of a run-on.